examples/src/test/scala/com/gu/tableversions/examples/DatePartitionedTableLoaderSpec.scala

```diff
@@ -25,7 +39,8 @@ class DatePartitionedTableLoaderSpec extends FlatSpec with Matchers with SparkHi
       Pageview("user-3", "sport/football", Timestamp.valueOf("2019-03-13 21:00:00"))
     )
 
-    loader.insert(pageviewsDay1.toDS().coalesce(2))
+    val userId = UserId("test user")
+    loader.insert(pageviewsDay1.toDS().coalesce(2), userId, "Day 1 initial commit")
```

Comment:
Why are you using `coalesce` here?

Reply:
I just wanted to limit the number of partitions in the tests data. It might be pointless, I'll try removing it.